src/Compilers/Core/Portable/RuleSet/RuleSetInclude.cs

```diff
@@ -80,6 +80,11 @@ private string GetIncludePath(RuleSet parent)
             List<string> found = new List<string>();
             string expandedPath = PortableShim.Environment.ExpandEnvironmentVariables(_includePath);
 
+            if (PathUtilities.IsUnixLikePlatform)
+            {
+                expandedPath = expandedPath.Replace('\\', PortableShim.Path.DirectorySeparatorChar);
```

Comment:
No, it isn't, which causes include file path resolution to fail on Linux/mac plaforms. Either the ruleset file author needs to fix their ruleset OR we have to do this replacement when parsing the ruleset includes.